deepfence_server/model/cloud_node.go

```diff
@@ -137,10 +137,11 @@ type CloudComplianceBenchmark struct {
 }
 
 type CloudComplianceScanDetails struct {
-	ScanId     string                     `json:"scan_id"`
-	ScanTypes  []string                   `json:"scan_types"`
-	AccountId  string                     `json:"account_id"`
-	Benchmarks []CloudComplianceBenchmark `json:"benchmarks"`
+	ScanId        string                     `json:"scan_id"`
+	ScanTypes     []string                   `json:"scan_types"`
+	AccountId     string                     `json:"account_id"`
+	Benchmarks    []CloudComplianceBenchmark `json:"benchmarks"`
+	StopRequested bool                       `json:"stop_requested"`
```

Comment:
As per previous comment, we don't need that flag
The status of the scan (CANCELLING) is already giving the information

Reply:
Please see the reason below:
https://github.com/deepfence/ThreatMapper/pull/1585#discussion_r1329605447

Reply:
Removed this flag as we will be making change in how we ingest the statuses